src/main/java/org/gridsuite/modification/server/modifications/ModificationUtils.java

```diff
@@ -722,25 +723,38 @@ private void applyModifications(ConnectablePosition.Feeder feeder,
 
     private void modifyFeederAdder(ConnectablePositionAdder.FeederAdder<?> feeder,
                                    Object modificationInfos,
+                                   Network network,
                                    List<ReportNode> reports,
                                    int feederNumber) {
+        var connectionName = getConnectionName(modificationInfos, feederNumber);
+        var connectionDirection = getConnectionDirection(modificationInfos, feederNumber);
+        var connectionPosition = getConnectionPosition(modificationInfos, feederNumber);
+        var equipmentId = getEquipmentId(modificationInfos);
```

Comment:
Here you don't really have an equipmentId, the type is AttributeModification<String>. You should remove the var if it brings fuzziness. Same for other vars